svg/_mixins.py

```diff
@@ -115,10 +115,10 @@ class ComponentTransferFunction(AttrsMixin):
 class Animation(AttrsMixin):
     # Animation value attributes
     calcMode: Literal["discrete", "linear", "paced", "spline"] | None = None
-    values: str | None = None
-    keyTimes: str | None = None
-    keySplines: str | None = None
-    keyPoints: str | None = None
+    values: str |  SemicolonSeperatedList | None = None
```

Comment:
This is equivalent to `SemicolonSeparatedList[Any]`. Looking at the spec, it says that each individual value is parsed as `href` for which the type is defined as `Name` which is, essentially, as string. So, if I read it right, it should be `SemicolonSeparatedList[str]`